brucemakallan/StackOverflow-Lite-React
# #162827256 View all questions and their answers

#### What does this PR do?
Adds functionality for getting all questions and their answers
#### Description of Task to be completed?
- Fetch all questions and their corresponding answers from the API database
- Show all questions and their answers in the UI
#### How should this be manually tested?
- Go to the homepage
- You should be able to view questions and their answers if logged in
#### Any background context you want to provide?
- You must be a registered user to view questions and answers
#### What are the relevant pivotal tracker stories?
[#162827256](https://www.pivotaltracker.com/story/show/162827256)
#### Screenshots
*Homepage showing Questions and their Answers*
![image](https://user-images.githubusercontent.com/36793346/50512619-1da86400-0aa4-11e9-9527-bf81e2a0a74b.png)

![image](https://user-images.githubusercontent.com/36793346/50512602-ff426880-0aa3-11e9-9eaa-de9d6ab6d159.png)
